released/packages/coq-dijkstra/coq-dijkstra.0.1.0/opam

```diff
@@ -26,6 +26,14 @@ build: [
   ]
 ]
 dev-repo: "git+https://github.com/fetburner/coq-dijkstra.git"
+tags: [
+  "keyword:graph theory"
+  "keyword:shortest path"
+  "keyword:Dijkstra's algorithm"
+  "category:Graph Theory"
```

Comment:
Note that there us no such category, but there is a category `"category:Computer Science/Graph Theory"`. See https://coq.inria.fr/opam/www/ for the current list. (Hover above a category to see its actual name.) Make also sure to not duplicate keywords, but as far as I can tell, this is fine.